src/diff.js

```diff
@@ -0,0 +1,1086 @@
+/*istanbul ignore next */
+// jscs:disable maximumLineLength
```

Comment:
We concatenate source files, so anything that gets disabled has to be enabled again for other files to still be validated.